source/features/copy-file.tsx

```diff
@@ -6,11 +6,11 @@ import {groupSiblings} from '../libs/group-buttons';
 
 function init() {
 	// This selector skips binaries + markdowns with code
-	for (const code of select.all('.file .blob-wrapper > .highlight:not(.rgh-copy-file)')) {
+	for (const code of select.all<HTMLInputElement>('.file .blob-wrapper > .highlight:not(.rgh-copy-file)')) {
```

Comment:
I'm guessing this is an `<input type="file" />`?

Reply:
No it's probably a regular element, I'd stick to `HTMLElement`